### PR TITLE
Broker and trigger types UTs

### PR DESCRIPTION
**Pull request by nachocano**
## Proposed Changes

- Adding UTs to broker and trigger types

**Release Note**

```release-note
NONE
```

### CODE REVIEW

**Comment by nachocano** [comment]
@Harwayne ready to review...

File: pkg/apis/eventing/v1alpha1/broker_types_test.go [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	duckv1alpha1 "github.com/knative/pkg/apis/duck/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var brokerConditionReady = duckv1alpha1.Condition{
```
Comment (Harwayne) [inline pkg/apis/eventing/v1alpha1/broker_types_test.go]: Put all of these in a single `var` block:

```golang
var(
  brokerConditionReady = ...
  brokerConditionIngress = ...
  ...
)
```

File: pkg/apis/eventing/v1alpha1/trigger_types_test.go [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	duckv1alpha1 "github.com/knative/pkg/apis/duck/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var triggerConditionReady = duckv1alpha1.Condition{
```
Comment (Harwayne) [inline pkg/apis/eventing/v1alpha1/trigger_types_test.go]: Put all of these in a single `var` block:

```go
var(
  triggerConditionReady = ...
  triggerConditionBrokerExists = ...
  ...
)
```

**Comment by Harwayne** [comment]
Ideally we would also have tests for validation and defaulting. But those can always be in subsequent PRs.

File: pkg/apis/eventing/v1alpha1/broker_types_test.go (hunk 1 above)
Comment (nachocano) [reply pkg/apis/eventing/v1alpha1/broker_types_test.go]: Done

File: pkg/apis/eventing/v1alpha1/trigger_types_test.go (hunk 2 above)
Comment (nachocano) [reply pkg/apis/eventing/v1alpha1/trigger_types_test.go]: Done

**Comment by nachocano** [comment]
> Ideally we would also have tests for validation and defaulting. But those can always be in subsequent PRs.

Added the trigger defaults and validation here. 
I think we still need to add the implementation to the broker validation.

**Comment by nachocano** [comment]
@Harwayne ready to review

**Pull request merged**